src/content/components/FeatureView/FeatureView.js

```diff
@@ -310,121 +426,193 @@ export class FeatureView extends React.PureComponent {
+        qm.runCachedQueries(
           allBugsQuery(subMetas),
           predicate,
-          ({ rsp: { bugs }, awaitingNetwork }) =>
-            this.setState({
-              bugs,
-              subMetas,
-              loaded: true,
-              awaitingNetwork,
-            })
+          ({ rsp: { bugs }, awaitingNetwork: newAwaitingNetwork }) => {
+            const result = {
+              nightlyResolved: [],
+              betaResolved: [],
+              releaseResolved: [],
+              resolved: [],
+            };
+            for (const bug of bugs) {
+              if (isBugResolved(bug)) {
+                if (["fixed", "verified"].includes(bug.cf_status_beta)) {
+                  result.betaResolved.push(bug);
+                } else if (
+                  ["fixed", "verified"].includes(bug.cf_status_nightly)
+                ) {
+                  result.nightlyResolved.push(bug);
+                } else if ("---" === bug.target_milestone) {
+                  result.resolved.push(bug);
+                } else {
+                  result.releaseResolved.push(bug);
+                }
+              }
+            }
+            setLoaded(true);
+            setAwaitingNetwork(newAwaitingNetwork);
+            setBugs(result);
+          }
         )
     );
-  }
-
-  bulkEditAll(e) {
-    const bugs = [...document.querySelectorAll("input[data-bug-id]")]
-      .map(i => i.value)
-      .filter(v => parseInt(v, 10));
-    e.target.href = `https://bugzilla.mozilla.org/buglist.cgi?bug_id=${bugs.join(
-      ","
-    )}&order=bug_id&tweak=1`;
-  }
-
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.match.params.id !== this.props.match.params.id) {
-      this.getBugs(nextProps.match.params.id);
-    }
-  }
+    return () => {
+      isMounted = false;
+    };
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+  return loaded ? (
+    <React.Fragment>
+      <FeatureBugList
+        title="Resolved in Nightly"
+        hideIfEmpty={false}
+        crossOutResolved={false}
+        bugs={bugs.nightlyResolved}
+        columns={[
+          ...resolvedDisplayColumns,
+          "cf_status_nightly",
+          "cf_status_beta",
+        ]}
+      />
+      <FeatureBugList
+        title="Resolved in Beta"
+        hideIfEmpty={true}
+        crossOutResolved={false}
+        bugs={bugs.betaResolved}
+        columns={[
+          ...resolvedDisplayColumns,
+          "cf_status_nightly",
+          "cf_status_beta",
+        ]}
+      />
+      <FeatureBugList
+        title="Resolved in Release"
+        hideIfEmpty={true}
+        crossOutResolved={false}
+        bugs={bugs.releaseResolved}
+        columns={[...resolvedDisplayColumns, "target_milestone"]}
+      />
+      <FeatureBugList
+        title="Other"
+        hideIfEmpty={true}
+        crossOutResolved={false}
+        bugs={bugs.resolved}
+        columns={[...resolvedDisplayColumns, "target_milestone"]}
```

Comment:
For Other section we can use `cf_fx_iteration`  in place of `target_milestone` which stays `---` for Other section by changing Line 380 above to use `displayColumnFilter`

```
  const resolvedFieldFilter = f =>
    !["cf_fx_points"].includes(f);

  const displayColumnFilter = f =>
    !["cf_fx_points", "cf_fx_iteration"].includes(f);
  
const resolvedDisplayColumns = useMemo(
    () => displayColumns.filter(displayColumnFilter),
    []
  );
```